packages/webpack/src/buildtime/policyGenerator.js

```diff
@@ -95,73 +90,57 @@ module.exports = {
-          )
-          diag.rawDebug(2, { skippingInspectingModule: module })
-          return
-        }
-        const packageName = getPackageNameForModulePath(
-          canonicalNameMap,
-          module.userRequest
-        )
-        const moduleRecord = new LavamoatModuleRecord({
-          // Knowing the actual specifier is not relevant here, they're used as unique identifiers that match between here and dependencies
-          specifier: module.userRequest,
-          file: module.userRequest,
-          type: isBuiltin(module.userRequest) ? 'builtin' : 'js',
-          packageName,
-          content: module.originalSource()?.source()?.toString(),
-          importMap: {
-            // connections are a much better source of information than module.dependencies which contain
-            // all imported references separately along with exports and fluff
-            ...Array.from(connections).reduce((acc, dep) => {
-              // @ts-expect-error - bad types?
-              const depSpecifier = dep.resolvedModule.userRequest
-              acc[depSpecifier] = depSpecifier
-              return acc
-            }, /** @type {Record<string, string>} */ ({})),
-          },
-          //ast: module._ast, - would have to translate to babel anyway
-        })
+    for (const { module, connections } of modulesToInspect) {
+      // Skip modules the user intentionally excludes.
+      // This is policy generation so we don't need to protect ourselves from an attack where the module has a loader defined in the specifier.
+      if (isExcludedUnsafe(module)) continue
+
+      if (module.userRequest === undefined) {
+        diag.rawDebug(1, `LavaMoatPlugin: Module ${module} has no userRequest`)
+        diag.rawDebug(2, { skippingInspectingModule: module })
+        continue
+      }
 
-        try {
-          moduleInspector.inspectModule(moduleRecord)
-        } catch (/** @type any */ e) {
-          throw new Error(
-            `LavaMoatPlugin: Failed to inspect module ${module.userRequest} for policy generation:\n ${e.message}\n If the file is not intended to be valid JavaScript, consider excluding it using LavaMoat.exclude loader.`,
-            { cause: e }
-          )
-        }
-      },
-      getPolicy: () => {
-        const policy = moduleInspector.generatePolicy({})
-        mkdirSync(location, { recursive: true })
-        writeFileSync(
-          path.join(location, 'policy.json'),
-          jsonStringifySortedPolicy(policy),
-          'utf8'
+      const packageName = getPackageNameForModulePath(
+        canonicalNameMap,
+        module.userRequest
+      )
+      const moduleRecord = new LavamoatModuleRecord({
+        // Knowing the actual specifier is not relevant here, they're used as unique identifiers that match between here and dependencies
+        specifier: module.userRequest,
+        file: module.userRequest,
+        type: isBuiltin(module.userRequest) ? 'builtin' : 'js',
```

Comment:
is this `isBuiltin` business for bundling Node.js projects, or...? why do we care if they are builtins on the Node.js side if they are replaced with shims during bundling?